mmeds/database/database.py

```diff
@@ -967,9 +968,41 @@ def check_study_name(self, study_name):
         """ Verifies the provided study name is valid and not already in use. """
         if not study_name.replace('_', '').isalnum():
             raise StudyNameError("Only alpha numeric characters and '_' are allowed in the study name")
-        if MMEDSDoc.objects(study_name=study_name):
+        if MMEDSDoc.objects(study_name=study_name, doc_type='study'):
             raise StudyNameError(f"Study name {study_name} already in use")
 
+    def check_sequencing_run_name(self, run_name):
+        """ Verifies the provided sequencing run name is valid and not already in use. """
+        if not run_name.replace('_', '').isalnum():
```

Comment:
Not a good idea to do replaces on the fly. In general, naming conventions should all happen in the same place, have code for handling IO specifically.